installing/install_quick_roxctl.adoc

```diff
@@ -0,0 +1,58 @@
+[id="install_quick_roxctl"]
+= Installing quickly by using the `roxctl` CLI
+include::modules/common-attributes.adoc[]
+:context: install_quick_roxctl
+
+toc::[]
+
+[role="_abstract"]
+{product-title} installs a set services on your {ocp} cluster.
+This topic describes the installation procedure for installing {product-title} on your {ocp} cluster by using the `roxctl` CLI.
+
+[NOTE]
+====
+For production environments, Red Hat recommends xref:../installing/installing_helm/install-helm-customization.adoc#installing-with-customizations-using-helm-charts[Installing {product-title} by using Helm charts].
+====
+
+High-level installation flow:
+
+. Install the `roxctl` CLI.
+. Use the `roxctl` CLI interactive installer to install the xref:../architecture/index.adoc#centralized-components_acs-architecture[centralized components] (Central and Scanner).
+. Install a Sensor to monitor your cluster.
+
+Before you install:
+
+* Understand xref:../architecture/index.adoc#acs-architecture_acs-architecture[{product-title} architecture].
+* Review the xref:prerequisites.adoc#acs-general-requirements_acs-prerequisites[prerequisites for installing {product-title}].
```

Comment:
Technically we're supposed to back up the whole way to the top, even if it's in the same directory, so: `../installing/prerequisites.adoc ...`